lib/ohai/plugins/linux/memory.rb

```diff
@@ -94,6 +95,10 @@
         memory[:hugepages][:surplus] = $1.to_s
       when /^Hugepagesize:\s+(\d+) (.+)$/
         memory[:hugepage_size] = "#{$1}#{$2}"
+      when /^Hugetlb:\s+(\d+) (.+)$/
+        memory[:hugetlb] = "#{$1}#{$2}"
+      when /^DirectMap([0-9][a-zA-Z]):\s+(\d+) (.+)$/
```

Comment:
that wouldn't match `DirectMap16k` you might want to use `[0-9]+` here for future proofing?